src/kit.ts

```diff
@@ -742,9 +742,16 @@ export async function scanForVSKits(pr?: ProgressReporter): Promise<Kit[]> {
     const prs = installs.map(async (inst): Promise<Kit[]> => {
         const ret = [] as Kit[];
         const targetArches: string[] = ['x86', 'x64', 'arm', 'arm64'];
+        const version = util.tryParseVersion(inst.installationVersion);
 
         const sub_prs: Promise<Kit | null>[] = [];
         MSVC_HOST_ARCHES.forEach(hostArch => {
+            if (hostArch === 'ARM64' && !(version && version.major >= 17 && version.minor >= 4)) {
```

Comment:
This condition should also succeed if `version.major > 17` or else 18.0-18.3 will inevitably fail in the future.

Reply:
Sorry, one last comment here, we should probably also disallow host=ARM64 if you're not on an ARM64 machine.  I guess we're not technically disabling x64 on x86 machines either, but I'm more concerned about adding unusable kits on x64 which has more users than ARM64 right now.

Reply:
Ah, the second point is interesting, as ARM64 can act as x86 and x64 host, but only ARM64 can be an arm64 host 🤣, so good point.

Reply:
I think I've fixed the conditional 🤔, the minor version for some reason makes it confusing 😅.